src/GoTrueApi.ts

```diff
@@ -76,6 +77,11 @@ export default class GoTrueApi {
     if (options?.scopes) {
       urlParams.push(`scopes=${encodeURIComponent(options.scopes)}`)
     }
+    if(options?.queryParams) {
+      for(const [param, value] of Object.entries(options.queryParams)) {
+        urlParams.push(`${encodeURIComponent(param)}=${encodeURIComponent(value)}`)
```

Comment:
Consider using URLSearchParams here. Since realtime uses it [here](https://github.com/supabase/realtime-js/blob/05d098c228d4303151d58448bf3e05cfb1c77225/src/RealtimeClient.ts#L451), I don't think compatibility across different environments should be an issue

Reply:
Makes sense, will use URLSearchParams